### DISCUSSION TITLE
Blitzwolf BW-SHP13 plug went wrong when I wrote endpoint

### DISCUSSION THREAD

**Discussion by sdalee** [Koenkk/zigbee2mqtt#13839] [Devices]
I tried to reset the (sum) energy using the Dev console (as described on the TuYa TS011F_plug_1 z2m device page):
Endpoint: 1
Cluster: 0x00
Command: 0

The energy was resetting, but Power, Current, Voltage reporting disappeared. See the z2m log:

Info 2022-09-03 22:00:21MQTT publish: topic 'zigbee2mqtt/Prb_Blitzwolf_Dugalj', payload '{"current":null,"energy":0.03,"indicator_mode":"off/on","linkquality":102,"power":null,"power_outage_memory":"restore","state":"ON","voltage":null}'

What could be the problem?

**Comment by kkossev** [comment]
Pair the plug again to z2m.

**Reply by sdalee** [reply]
I tried this one first. No change, same result, no voltage, no current, no power. :-(

**Reply by kkossev** [reply]
Did you try removing/deleting the plug from z2m and then pairing it again as a new device? 
...just a suggestion, I have not personally tried this command on any of my devices. It's a kind of 'factory reset' command for Tuya devices, I remember there were other users saying that their plug become inoperative after missing this command.

**Reply by sdalee** [reply]
Yes, I used 'force remove' and new pairing as a new device.

Unfortunately, I think I was like the 'other users' I hope there is a solution to this problem.
